### PR TITLE
Replace sscanf by equivalent native PHP functions because sscanf can be disabled for security reasons.

### PR DESCRIPTION
**Pull request by niconoe-**
As sscanf can be disabled for security reasons, as mentioned in #270 , and also explained here: https://www.php.net/manual/en/function.sscanf.php#49713, I changed the usages by other native PHP functions, so performance should not be impacted.

Fixes: #270

### CODE REVIEW

File: src/Token.php [hunk 1]
```diff
@@ -252,17 +252,16 @@ public function extract()
                 if ($this->flags & self::FLAG_NUMBER_HEX) {
                     if ($this->flags & self::FLAG_NUMBER_NEGATIVE) {
                         $ret = str_replace('-', '', $this->token);
-                        sscanf($ret, '%x', $ret);
-                        $ret = -$ret;
+                        $ret = -hexdec($ret);
```
Comment (williamdes) [inline src/Token.php]:
```suggestion
                        $ret -= hexdec($ret);
```
Would this be more obvious?

**Comment by williamdes** [comment]
Tests seem to fail https://travis-ci.org/phpmyadmin/sql-parser/jobs/644946170#L323

File: src/Token.php (hunk 1 above)
Comment (niconoe-) [reply src/Token.php]: I don't think it's the same behavior. If, for example, `$this->token` worth `-a0` (-16 in decimal), expected value for `$ret` is -16. If you do `$ret -= hexdec($ret)`, you're doing `$ret = a0 - 16`, so basically, `$ret = 0`.

Comment (williamdes) [reply src/Token.php]: Yes, I did not read it right
Can you add a test so we are sure nobody removes the sign by mistake?

**Comment by williamdes** [comment]
Can you run `./vendor/bin/phpcbf` and commit the coding standard fixes
The PR will be okay after that

**Comment by williamdes** [comment]
Thank you @niconoe- that will reduce errors being reported to our server :tada:

**Pull request merged**